docs/rule/attributes-order.md

```diff
@@ -0,0 +1,59 @@
+# attributes-order
+
+Element attributes must be ordered. By default, the order is:
+
+- arguments
+- attributes
+- element modifiers
+- ...attributes
```

Comment:
Splattributes shouldn't be mentioned in the ordering anymore right? Since we allow the user to place the splattributes wherever they like now from my understanding.